lib/eventFactory.js

```diff
@@ -206,7 +207,9 @@ function createBuilds(config) {
         .then((jobsToStart) => {
             // No jobs to start (eg: when jobs are disabled or startFrom is not valid)
             if (jobsToStart.length === 0) {
-                throw new Error('No jobs to start');
+                winston.info(`No jobs to start in event ${eventId}.`);
```

Comment:
So what happens if all jobs are disabled and user clicks start? Previously they will see the error message. We should have provide some feedback to users still right?

Reply:
Looks good 👍